app/src/main/java/com/pokemonshowdown/data/ItemDex.java

```diff
@@ -15,17 +16,34 @@
 import java.io.InputStreamReader;
 import java.util.HashMap;
 import java.util.Iterator;
+import java.util.Map;
 
 public class ItemDex {
```

Comment:
Now that we don't need to read the Json file anymore I think we should change this class into a Utility class instead of a Singleton. Which mean we don't save context and we don't keep mItemDexEntries in memory anymore.